apps/storybook/src/MessageLayout.stories.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
```

Comment:
made react-components export structure and story structure more consistent with rest of app